var/spack/repos/builtin/packages/googletest/package.py

```diff
@@ -14,12 +14,10 @@ class Googletest(CMakePackage):
     git = "https://github.com/google/googletest"
 
     version("main", branch="main")
+    version("1.12.1", sha256="81964fe578e9bd7c94dfdb09c8e4d6e6759e19967e397dbea48d1c10e45d0df2")
+    version("1.12.0", sha256="2a4f11dce6188b256f3650061525d0fe352069e5c162452818efbbf8d0b5fe1c")
     version("1.11.0", sha256="07b0896360f8e14414a8419e35515da0be085c5b4547c914ab8f4684ef0a3a8e")
-    version(
-        "1.10.0",
-        sha256="e4a7cd97c903818abe7ddb129db9c41cc9fd9e2ded654be57ced26d45c72e4c9",
-        preferred=True,
```

Comment:
@sethrj it looks like you added this in #24290 because it seemed like 1.11.0 broke some packages. Unfortunately, `@1.10.0+gmock` doesn't build for me (at least on macOS arm64), so I'm not sure if 1.10.0 is a good default either. I think it's better to default to the latest version and fix any packages that are broken in CI.

Reply:
This was probably "solved" by using clingo instead of the old concretizer: I remember the newer versions causing E4S some issues, but either their code has been upgraded or conflicts have been added.